.github/workflows/prerelease.yml

```diff
@@ -79,12 +79,25 @@ jobs:
           aws-region: ${{ env.BUILD_REGION }}
           role-to-assume: ${{ secrets.BUILD_ROLE_TO_ASSUME_STAGING }}
           role-duration-seconds: 3600
+      # Ansible will find /usr/bin/python3 and use it, so we must
```

Comment:
For additional clarity, I think this comment and the one in `release.yml` should explicitly reference packer (in addition to Ansible).

Reply:
See commit 046ca10.